pkgs/top-level/all-packages.nix

```diff
@@ -8127,8 +8127,6 @@ with pkgs;
   libc =
     let
       inherit (stdenv.hostPlatform) libc;
-      # libc is hackily often used from the previous stage. This `or`
-      # hack fixes the hack, *sigh*.
```

Comment:
Why remove this comment?

Reply:
IIRC this comment was referring to how everything was `libcCross or targetPackages.libc` here. That was removed recently when libcCross was removed. There's no `or` hack to fix any hack so I removed it. Did I understand the comment wrong?

Reply:
I think you're right, but it shouldn't be in a commit called " lib.systems: add mlibc as a libc option".